roslibrust/src/ros1/subscriber.rs

```diff
@@ -39,6 +39,29 @@ impl<T: RosMessageType> Subscriber<T> {
     }
 }
 
+pub struct SubscriberAny {
+    receiver: broadcast::Receiver<Vec<u8>>,
+    _phantom: PhantomData<Vec<u8>>,
+}
+
+impl SubscriberAny {
```

Comment:
If we go with this approach, I think we also need to implement Drop for SubscriberAny matching what Subscriber::drop() does to un-subscribe under the hood.

Reply:
May be worth duplicating the subscriber dropping integration test for SubscriberAny to make sure behavior stays the same.

Reply:
Whoops sorry, lost my mind for a second!

For ros1, dropping the last tx side of the channel causes subscription clean-up. So this is good!